renderdoc/driver/d3d12/d3d12_device_wrap.cpp

```diff
@@ -660,6 +660,179 @@ bool WrappedID3D12Device::Serialise_CreateCommittedResource(
   return true;
 }
 
+bool WrappedID3D12Device::Serialise_CreatePlacedResource(ID3D12Heap *pHeap, UINT64 HeapOffset,
```

Comment:
The `Serialise_` functions should always be placed immediately above the function they're serialising.